src/styles/index.js

```diff
@@ -19,9 +19,10 @@ const colors = {
 
 // Returns an OpenLayers Style for a given color.
 const colorStyles = (color) => {
-  const rgba = colors[color] ? colors[color] : colors.yellow;
+  color = color.startsWith('#') || color.startsWith('rgb') ? color : colors[color] || colors.yellow;
```

Comment:
It's a mildly breaking change (i.e. unrecognized, but non-empty inputs would start getting passed through instead of coerced to yellow), but I think we should consider;

```suggestion
  if (!color) {
    color = colors.yellow;
  } else if (colors[color]) {
    color = colors[color];
  }
```

What do you think @mstenta?

Reply:
We looked at this on the dev call and the consensus was that we can move forward with the slightly backwards-incompatible change.